t/websocket.t

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+
+use Test::More;
+use Test::Fatal;
+use IO::Socket::INET;
+use IO::Async::Test;
+use IO::Async::Loop;
+use Net::Async::WebSocket::Server;
+use Net::Async::Blockchain::Client::Websocket;
+use JSON::MaybeUTF8 qw(encode_json_utf8 decode_json_utf8);
+
+my $loop = IO::Async::Loop->new;
+testing_loop($loop);
+
+my $serversock = IO::Socket::INET->new(
```

Comment:
Use [`IO::Socket::IP`](http://metacpan.org/pod/IO::Socket::IP) instead for new code - but I don't think this is needed, `$server->listen(service => 0)` should be enough?